config/params.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'yiisoft/form' => [
+        'bulma' => [
+            'enabled' => true,
```

Comment:
What's the use case for disabling it?

Reply:
Multiple packages can be installed (yii-bulma for example). This makes sure the unwanted changes can be turned off.

Reply:
@terabytesoftw insisted of it being able to be disabled